localstack/utils/testutil.py

```diff
@@ -50,52 +52,61 @@ def create_dynamodb_table(table_name, partition_key, env=None, stream_view_type=
     return table
 
 
-def create_lambda_archive(script, stream=None, get_content=False, libs=[], runtime=None):
+def create_lambda_archive(script, get_content=False, libs=[], runtime=None):
     """Utility method to create a Lambda function archive"""
     tmp_dir = tempfile.mkdtemp(prefix=ARCHIVE_DIR_PREFIX)
     TMP_FILES.append(tmp_dir)
     file_name = get_handler_file_from_name(LAMBDA_DEFAULT_HANDLER, runtime=runtime)
-    script_file = '%s/%s' % (tmp_dir, file_name)
+    script_file = os.path.join(tmp_dir, file_name)
     save_file(script_file, script)
     # copy libs
     for lib in libs:
         paths = [lib, '%s.py' % lib]
         target_dir = tmp_dir
-        root_folder = '%s/lib/python*/site-packages' % LOCALSTACK_VENV_FOLDER
+        root_folder = os.path.join(LOCALSTACK_VENV_FOLDER, 'lib/python*/site-packages')
         if lib == 'localstack':
             paths = ['localstack/*.py', 'localstack/utils']
             root_folder = LOCALSTACK_ROOT_FOLDER
-            target_dir = '%s/%s/' % (tmp_dir, lib)
+            target_dir = os.path.join(tmp_dir, lib)
             mkdir(target_dir)
         for path in paths:
-            file_to_copy = '%s/%s' % (root_folder, path)
+            file_to_copy = os.path.join(root_folder, path)
             for file_path in glob.glob(file_to_copy):
-                run('cp -r %s %s/' % (file_path, target_dir))
+                name = os.path.join(target_dir, file_path.split('/')[-1])
+                if os.path.isdir(file_path):
+                    shutil.copytree(file_path, name)
+                else:
+                    shutil.copyfile(file_path, name)
 
     # create zip file
-    return create_zip_file(tmp_dir, get_content=True)
+    return create_zip_file(tmp_dir, get_content=get_content)
 
 
-# TODO: Refactor this method and use built-in file operations instead of shell commands
-def create_zip_file(file_path, include='*', get_content=False):
+def create_zip_file(file_path, get_content=False):
```

Comment:
Looks like `create_zip_file(..)` is only used from this file, so it should be safe to remove the `include` parameter here. 👍